MIGRATION.md

```diff
@@ -0,0 +1,82 @@
+Migrating to @oclif/core
+==============
+
+Migrating to `@oclif/core` from the old oclif libraries (`@oclif/config`, `@oclif/command`, `@oclif/error`, `@oclif/parser`) is relatively straight forward.
+
+## Update Imports
+
+Any imports from the old libraries can be replaced with `@oclif/core`. For example,
+
+```typescript
+import Help from '@oclif/plugin-help';
+import {Topic} from '@oclif/config';
+import {Command, flags} from '@oclif/command'
+```
+
+Can be changed to:
```

Comment:
```suggestion
Changes to:
```

Reply:
FYI, modals (like "can") are often confusing because people aren't sure if they're supposed to follow the directions or not.  So I suggested prescriptive language (i.e. "do this").  Okay?